src/Media.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import elementType from 'react-prop-types/lib/elementType';
+import classNames from 'classnames';
+
+import Heading from './MediaHeading';
+import Body from './MediaBody';
+import Left from './MediaLeft';
+import Right from './MediaRight';
+import List from './MediaList';
+import ListItem from './MediaListItem';
+
+const Media = React.createClass({
+  displayName: 'Media.Media',
```

Comment:
Probably call this `Media` and make this the default export. In other words, do e.g.

```js
Object.assign(Media, { Heading, Body, ... });

export default Media;
export { Heading, Body, ... }; // Optional.
```